wolfssl/wolfcrypt/asn.h

```diff
@@ -780,6 +780,14 @@ extern const WOLFSSL_ObjectInfo wolfssl_object_info[];
     #define WOLFSSL_TLS_FEATURE_SUM 92
 #endif
 
+#ifndef WOLFSSL_MAX_ALT_NAMES
```

Comment:
Document inline comment what this limits.

Reply:
done.